docs/project/governance.md

```diff
@@ -0,0 +1,97 @@
+Github to label and close issues. Their work is
+crucial to improve the communication in the project.
+
+Similarly to what has been decided in the [CPython project](
+https://devguide.python.org/triaging/#becoming-a-member-of-the-python-triage-team),
+any contributor may become a member of this team, after
+showing some continuity in participating in pyodide
+development (with pull requests and reviews).
+Any core developer is welcome to propose a pyodide contributor to join the
+community members team. Other core developers are then consulted: while it is expected
+that most acceptances will be unanimous, a two-thirds majority is enough.
+
+### Core developers
+
+Core developers are community members who have shown that they are dedicated to
+the continued development of the project through ongoing engagement with the
+community. They have shown they can be trusted to maintain pyodide with
+care. Being a core developer allows contributors to more easily carry on
+with their project related activities by giving them direct access to the
+project’s repository and is represented as being a member of the core team on the
+pyodide [GitHub organization](https://github.com/orgs/pyodide/teams/core/members).
+Core developers are expected to review code
+contributions, can merge approved pull requests, can cast votes for and against
+merging a pull-request, and can be involved in deciding major changes to the
+API.
+
+New core developers can be nominated by any existing core developers. Once they
+have been nominated, there will be a vote by the current core developers.
+Voting on new core developers is one of the few activities that takes place on
+the project's private communication channels. While it is expected that most votes
+will be unanimous, a two-thirds majority of the cast votes is enough. The vote
+needs to be open for at least 1 week.
+
+Core developers that have not contributed to the project (commits or GitHub
+comments) in the past 2 years will be asked if they want to become emeritus
+core developers and recant their commit and voting rights until they become
+active again.
+
+
+## Decision Making Process
+
+Decisions about the future of the project are made through discussion with all
+members of the community. All non-sensitive project management discussion takes
+place on the project contributors' [issue
+tracker](https://github.com/pyodide/pyodide/issues) and on [Github
+discussion](https://github.com/pyodide/pyodide/discussions).
+Occasionally, sensitive discussion occurs on a private communication channels.
+
+pyodide uses a "consensus seeking" process for making decisions. The group
+tries to find a resolution that has no open objections among core developers.
+At any point during the discussion, any core-developer can call for a vote,
+which will conclude two weeks from the call for the vote. This is what we
+hereafter may refer to as “the decision making process”.
```

Comment:
It's a little unclear to me what scenarios would require a formal vote, is it just for any issue that leads to a significant amount of contention?

Reply:
The only scenario where a formal note needs to happen in the current document is for adding new maintainers. All other points should ideally reach a consensus by discussion, and only fall back on voting if agreement cannot be reached and a decision needs to be taken.